dependencies/maven/artifacts.bzl

```diff
@@ -21,6 +21,7 @@ artifacts = [
     "io.github.microutils:kotlin-logging-jvm",
     "junit:junit",
     "org.antlr:antlr4-runtime",
+    "org.jetbrains.compose.compiler:compiler",
```

Comment:
We import the compose compiler in a more traditional way now, along with all of our other external dependencies.